### ISSUE TITLE
Might create blank line at start of file with comments

### ISSUE THREAD

**Issue by blueyed** [timothycrosley/isort#889]
Running `isort --diff` with the following will add a blank line at the start:
```python
# comment
import os
# comment2
import sys
```

```diff
@@ -1,4 +1,5 @@
+
+import os
 # comment
-import os
 # comment2
 import sys
```

The result then looks like this:
```python

import os
# comment
# comment2
import sys
```

**Comment by timothycrosley** [comment]
Thank you for reporting this!
Should be fixed in the latest release (4.3.13) of isort.

Thanks!

~Timothy

**Comment by blueyed** [comment]
Awesome, thanks!  Confirmed to be fixed with 855806f.
